guidelines.mdx

```diff
@@ -0,0 +1,520 @@
+
+| ✅ DO WRITE                                | ❌ DO NOT WRITE                                 |
+| ------------------------------------------ | ----------------------------------------------- |
+| The price is €0.11 per hour                | The price is €0,11 per hour <br/> The price is 0,11€ per hour <br/> The price is 0.11€ per hour              |
+| The maximum charge is €150,000             | The maximum charge is €150.000 <br/> The maximum charge is 150 000€ |
+
+### Dates and times
+
+- For times, use the 24-hour format. If it is necessary to use the 12-hour format, use `a.m.` or `p.m.` appropriately
+- For dates, use `day/month/year` or `DD/MM/YY`. 
+- You can also write out the date, e.g. ` 14 January 2022`. Avoid ordinals, e.g. `14th January`. For a full list of allowed date formats, see the [UX writing guide](https://ultraviolet.scaleway.com/6dd9b5c45/p/45cfa5-date-and-time/b/210929).
+
+### Bits and bytes
+
+- Add a space between the bits/bytes abbreviation and the number, e.g. `100 MB`, `1 Gbps`
+- Use the following abbreviations: 
+
+| Term                  | Abbreviation      | Example                                  |
+| --------------------- | ------------------|----------------------------------------- |
+| megabyte              | MB                | Memory: 100 MB                           |
+| gigabyte              | MB                | Memory: 1 GB                             |
+| gigiabit              | none              | Connectivity; up to 10 gigabit           |
+| gigabits per second   | Gbps              | Bandwidth: 1 Gbps                        |
+| megabits per second   | Mbps              | Bandwidth: 100 Mbps                      |
+| gigabute-seconds      | GB-s              | Storage: 1 GB-s                          |
+
+## Wording and vocabulary
+
+### Environment variables
+
+When citing Scaleway API or CLI commands, use the [established environment variable conventions](https://github.com/scaleway/scaleway-sdk-go/tree/master/scw#environment-variables).
+
+### Console wording conventions
+
+When referring to different aspects of the Scaleway console, use the following terms:
+
+| Console element             | Screenshot                                                     |
+|-----------------------------|----------------------------------------------------------------|
+| Side menu                   | <Lightbox src="scaleway-console-side-menu.webp" alt=""  />     |
+| Organization drop-down menu | <Lightbox src="scaleway-org-dropdown.webp" alt=""  />          |
+| $PRODUCT_NAME dashboard, e.g. Object Storage dashboard | <Lightbox src="scaleway-console-dashboard.webp" alt=""  />  |
+| $PRODUCT_NAME creation page ( NOT `splashscreen`), e.g. Instances creation page | <Lightbox src="scaleway-instances-creation-page.webp" alt=""  /> | 
+| $PRODUCT_NAME creation wizard, e.g. Load Balancer creation wizard | | <Lightbox src="scaleway-creation-wizard.webp" alt=""  /> |           
```

Comment:
is there supposed to be an extra one here?

Reply:
Well spotted! Thanks